homeassistant/components/incomfort/const.py

```diff
@@ -0,0 +1,3 @@
+"""Constants for `InComfort` integration."""
```

Comment:
I don't think we usually do markdown in docstrings

Reply:
I had some trouble with the spelling check that did not allow spelling it lowercase.

Reply:
I thing I have changed them all now